.buildkite/x-pack/pipeline.xpack.agentbeat.yml

```diff
@@ -60,26 +60,17 @@ steps:
 
       - label: ":linux: Agentbeat/Integration tests Linux"
         key: "agentbeat-it-linux"
-        depends_on:
-          - agentbeat-package-linux
         env:
           ASDF_NODEJS_VERSION: 18.17.1
           PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
           SNAPSHOT: true
         command: |
           set -euo pipefail
-          echo "~~~ Downloading artifacts"
-          buildkite-agent artifact download x-pack/agentbeat/build/distributions/** . --step 'agentbeat-package-linux'
-          ls -lah x-pack/agentbeat/build/distributions/
```

Comment:
I don't why we were doing this. For the integration test, test binaries will be created which is already handled in mage goIntegTest.